submit_cgap/utils.py

```diff
@@ -80,3 +88,71 @@ def script_catch_errors():
             show(ERROR_HERALD)
             show(f"{e.__class__.__name__}: {e}")
             exit(1)
+
+
+# TODO: If deemed generally useful then move to dcicutils.
+def check_repeatedly(check_function: Callable,
```

Comment:
Once again a useful function that probably belongs in `dcicutils` eventually. I could see this being useful as another thread in the ingester potentially.